### DISCUSSION TITLE
快捷键相关

### DISCUSSION THREAD

**Discussion by Neo-Ze** [the1812/Bilibili-Evolved#2129] [提问 / Q&A] [answered]
使用快捷键扩展后，番剧页面无法使用，只有ESC可用，空格暂停、F全屏、M静音等B站自带快捷键都无法使用，而在其他页面则正常。取消快捷键扩展B站播放器自带快捷键使用正常。
![Snipaste_2021-08-09_00-01-35](https://user-images.githubusercontent.com/81399833/128638147-a039f28e-ce57-49a5-97e3-a0be82624d9f.jpg)

**Comment by RingingVolcano** [comment, marked as answer]
重复：#2092
请善用搜索

不 要 急：[v1.12.13](https://github.com/the1812/Bilibili-Evolved/releases/tag/v1.12.13)
 > 仅做了一些小适配, 很多地方我也还没摸清楚
